src/panels/logbook/ha-logbook.ts

```diff
@@ -305,6 +306,7 @@ export class HaLogbook extends LitElement {
         this.entityIds,
         this.deviceIds
       );
+      this._subscribed = await subscribePromise;
```

Comment:
`_subscribed` is typed as promise, and it also awaited in the code, we now no longer save a `promise` in `_subscribed`.

Also now `_subscribed` is set later, so that when `_unsubscribe` is called, `_subscribed` might not be set yet and we never unsubscribe.